### ISSUE TITLE
apostrophes do not show correctly on my locally served site

### ISSUE THREAD

**Issue by abotzki** [elixir-europe-training/ELIXIR-TrP-ContainersPython-CodeRep#6]
Apostrophes are not rendered correctly on my locally served site - MacOS 
seems like a UTF-8 encoding issue

**Comment by penyuan** [comment]
Yeah I still can't find a way to fix this either. But I've created a Stackoverflow question [here](https://stackoverflow.com/q/76820242/186904). (let's see if someone uses ChatGPT to post an answer...)

**Comment by penyuan** [comment]
I've also posted [this issue](https://github.com/mkdocs/mkdocs/issues/3328) in the MKDocs repository.

**Comment by penyuan** [comment]
Issue solved [here](https://github.com/mkdocs/mkdocs/discussions/3329#discussioncomment-6618758)!!
